### DISCUSSION TITLE
How to set global disable display relative time?

### DISCUSSION THREAD

**Discussion by Tomorrowxxy** [community/community#109532] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How to set global disable display relative time?
And relative time really bothers me.

For example
![image](https://github.com/community/community/assets/33508761/05050021-0877-4971-acd8-69c611938763)
![image](https://github.com/community/community/assets/33508761/71886e4a-e27f-497f-b4be-1a883c583781)

Although I know that the mouse hover can display the format time, it is too inconvenient
I would like to know how to directly format the global time display to '2024-02-22 13:51:44' ?

**Comment by RobertKrawitz** [comment]
There has been a lot of discussion of this topic [here](https://github.com/orgs/community/discussions/5972).
